src/containers/Login/Greeting.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {Switch, Route} from 'react-router-dom';
+
+import { styled } from '../../styles';
+import { LoginLandscapeImage } from "../../assets";
+
+import { LoginForm } from "./LoginForm";
+
+const Wrapper = styled.div`
+  width: 1000px;
+  height: 600px;
+  display: flex;
+  box-shadow: 5px 5px 5px gray;
+  border: 1px solid;
```

Comment:
you can combine borders styles in one line
border: 1px solid ${({theme})=>theme.colors.black};